SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -319,7 +355,19 @@ class CFluidModel {
    * \brief Virtual member.
    * \param[in] T - Temperature value at the point.
    */
-  virtual void SetTDState_T(su2double val_Temperature, const su2double* val_scalars = nullptr) {}
+  virtual void SetTDState_T(su2double val_Temperature, const su2double* val_scalars = nullptr) { }
+
+  /*!
+   * \brief Virtual member.
+   * \param[in] val_scalar - Pointer to the transported scalars.
+   */
+  virtual unsigned long SetScalarSources(const su2double* val_scalars) { return 0; }
+
+  /*!
+   * \brief Virtual member.
+   * \param[in] val_scalar - Pointer to the transported scalars.
+   */
+  virtual unsigned long SetScalarLookups(const su2double* val_scalars) { return 0; }
```

Comment:
Put these with the other flamelet LUT functions and please don't use this mysterious `Virtual member` type of brief.